catalog.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Нёрдс - каталог</title>
+</head>
+<body>
+    <header>
+        <nav>
+          <a href="index.html">
+            <img src="https://via.placeholder.com/160x65" alt="Логотип компании Nerds">
+          </a>
+          <ul>
+            <li><a href="index.html">Студия</a></li>
+            <li><a href="#">Клиенты</a></li>
+            <li><a href="#">Магазин</a></li>
```

Comment:
```suggestion
            <li><a>Магазин</a></li>
```
Это текущая страница.